lib/cocoapods/installer/xcode/pods_project_generator.rb

```diff
@@ -92,6 +92,10 @@ def write
               installation_result.test_native_targets.each do |test_native_target|
                 scheme.add_test_target(test_native_target)
               end
+
+              installation_result.app_native_targets.each do |add_build_target|
```

Comment:
That's actually a good question. As of the current state of cocoapods with PR my thinking is no because of the fact there is still a target that will be defined as the main "app". 

I don't feel too strongly about that though and could be easily convinced